src/jubilation-line-chart/index.jsx

```diff
@@ -0,0 +1,62 @@
+// @flow
```

Comment:
This is mostly a straight copy of Scatterplot component.  Is there any merit to extracting an HOC for `Chart` that handles most of the imports, the common props, ctor, UUID logic, WillReceiveProps, and any others you think that merits?